roles/openshift_cluster_metrics_hawkular/files/cluster-metrics/enterprise-metrics.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: "v1"
```

Comment:
Instead of having the templates in here, the ones on disk should be used instead (openshift_examples). potentially we may need to land those templates even when the user chooses to skip the other templates.